package.json

```diff
@@ -2,10 +2,16 @@
   "$schema": "https://json.schemastore.org/package.json",
   "name": "saleor-app-payment-template",
   "description": "Saleor App Payment Template",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "private": true,
+  "repository": "github:saleor/saleor-app-payment-template",
+  "homepage": "https://github.com/saleor/saleor-app-payment-template",
+  "bugs": {
+    "url": "https://github.com/saleor/saleor-app-payment-template/issues",
+    "email": "hello@saleor.io"
+  },
   "scripts": {
-    "dev": "pnpm generate && NODE_OPTIONS='--inspect' next dev",
```

Comment:
Should we add a separate script for running Next.js with debug enabled?

Reply:
You can do `NODE_OPTIONS='--inspect' pnpm dev`, right?